src/fhirtypes/ElementDefinition.ts

```diff
@@ -98,7 +99,7 @@ export class ElementDefinitionType {
  * @see {@link http://hl7.org/fhir/R4/elementdefinition.html}
  */
 export class ElementDefinition {
-  private _id: string;
+  private _id: FHIRId;
```

Comment:
Technically, this is not correct.  `ElementDefinition` inherits the `id` field from `Element` -- which described `id` as a `string` (not an `id`):
![image](https://user-images.githubusercontent.com/2278253/73193071-0220ea00-40f8-11ea-8926-45beef7b55de.png)

It would actually be a significant problem if it was defined as an `id` because element ids can get very long and we'd definitely run into the character length limit on ids for deeply nested elements.

Reply:
Ah, thanks for catching this!